schema/image_description_schema.json

```diff
@@ -160,6 +164,10 @@
                         "image"
                     ],
                     "description": "The image file type"
+                },
+                "size": {
```

Comment:
nit: use `filesize` to be more explicit, instead of `size`?

Reply:
OK!

Reply:
Just need to update tests.